CDCgov/data-exchange-portal
# Portal 950/api oauth spike

Testing steps:
1. Download app dependencies with yarn install
2. Download API dependencies by running npm install in the /api subdirectory
3. Download and install the Azure CLI and Azure function CLI
4. Run the front end app with yarn dev
5. Build the backend azure function with `npm run build` in the /api subdirectory
6. In a separate terminal, run the api server with `func host start` 
7. Navigate to localhost:5173 in a browser to see the frontend app
8. Click the "login with sams" button
9. This should redirect you to the sams stg login page.  Sign in with username/password or AMS
10. Upon success, this should redirect you to another page and show an auth code string
